schemas/compat-data-schema.md

```diff
@@ -218,12 +218,14 @@ entirely unknown. Examples:
  "version_added": "3.5"
 }
 ```
-* Supported, but version unknown:
+* Supported, but earliest version unknown:
+  The version in the version range is the version at the time of commit.
 ```json
 {
-  "version_added": true
+  "version_added": "<=60"
 }
 ```
+  It’s also possible to use `true`, but this is deprecated in favour of version ranges.
```

Comment:
This is definitely just a nitpick, and this does not affect my review of this PR.  It's mostly a curiosity thing.  In this repository (or for MDN as a whole), have we established whether we use American or European spelling?  If not, I think that we should probably figure out which one to use and stick with it for consistency purposes.  (CC @Elchi3)